package.json

```diff
@@ -29,7 +29,7 @@
     "docs": "pnpm --filter rolldown-docs run dev",
     "docs:build": "pnpm --filter rolldown-docs run build",
     "docs:preview": "pnpm --filter rolldown-docs run preview",
-    "version": "yarn lerna version --force-publish --no-private --no-git-tag-version && node scripts/post-version.cjs",
+    "version": "pnpm changeset version",
```

Comment:
we should maybe remove this since we have it in the `justfile`.

Reply:
@hyf0 What do you think？

Reply:
Yeah. I agree with @milesj. He knows this better.

Reply:
Ok, Now I remove this script about version.